lib/core/src/examples/shapes.rs

```diff
@@ -24,6 +26,24 @@ pub fn run_example_shapes() {
 }
 
 fn init(world: &World) {
+    let container      = DomContainer::from_id("app").expect("Couldn't get container");
```

Comment:
I don't like that we create this manually. It should be hidden. Scene has a reference to dom object. Maybe it should use `DomContainer` internally. Anyway, user-facing code of the lib should not require this line.